src/resolvers/exotics/bitbucket-resolver.js

```diff
@@ -7,6 +7,20 @@ export default class BitbucketResolver extends HostedGitResolver {
   static hostname = 'bitbucket.org';
   static protocol = 'bitbucket';
 
+  static isVersion(pattern: string): boolean {
+    // bitbucket proto
+    if (pattern.startsWith('bitbucket:')) {
```

Comment:
This is already covered by `protocol` above, right? Please use the already defined constant or modify it to not use a custom string.